wordpress-mobile/WordPress-iOS
# [Gutenberg] Return the media type for uploads

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/pull/1378

To test:
Please check the related Gutenberg feature https://github.com/WordPress/gutenberg/pull/17537

Update release notes:
